standard/standard_4.php

```diff
@@ -728,7 +728,7 @@ function header_remove(?string $name): void {}
  * @return bool headers_sent will return false if no HTTP headers
  * have already been sent or true otherwise.
  */
-function headers_sent(&$filename, &$line): bool {}
+function headers_sent(string &$filename = null, int &$line = null): bool {}
```

Comment:
```suggestion
function headers_sent(&$filename = null, &$line = null): bool {}
```

Reply:
So can we add only the default values then?

Reply:
That is correct! Tests are green now